src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -98,6 +92,12 @@ def get(self, request: Request, organization) -> Response:
 
         top_trending_transactions = {}
 
+        experiment_use_project_id = features.has(
+            "organizations:performance-trendsv2-dev-only",
```

Comment:
just reusing this internal flag to gate this change from affecting existing trends